roles/kubernetes/node/defaults/main.yml

```diff
@@ -40,6 +40,11 @@ system_cpu_reserved: 500m
 system_master_memory_reserved: 256Mi
 system_master_cpu_reserved: 250m
 
+## Eviction Thresholds to avoid system OOMs
+# https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/#eviction-thresholds
+eviction_hard: {}
+eviction_hard_master: {}
```

Comment:
```suggestion
eviction_hard_control_plane: {}
```

Reply:
Please see https://www.cncf.io/announcements/2021/10/13/inclusive-naming-initiative-announces-new-community-resources-for-a-more-inclusive-future/ which kubernetes and related projects adhere to.

Reply:
thanks for the doc :ok_hand: I will modify it later should I also modify these vars in this pr :
```
system_master_cpu_reserved --> system_control_plane_cpu_reserved
system_master_memory_reserved --> system_control_plane_memory_reserved
kube_master_memory_reserved --> kube_control_plane_memory_reserved
kube_master_cpu_reserved --> kube_control_plane_cpu_reserved